src/test/java/org/influxdb/InfluxDBLogLevelTest.java

```diff
@@ -0,0 +1,53 @@
+package org.influxdb;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.influxdb.dto.Pong;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import net.jcip.annotations.NotThreadSafe;
+
+/**
+ * Test the InfluxDBimpl log level setting from system property.
+ *
+ * @author hoan.le [at] bonitoo.io
+ *
+ */
+@RunWith(JUnitPlatform.class)
+@NotThreadSafe
```

Comment:
Are you 100% sure this annotation will affect how JUnit works?
Anyway, instead of fixing the bad practice you introduced by making changes that may affect other tests you are here trying to do something that would make the tests not going in parallel? I don't get it.